{"uid":"72050e38879c10a3","name":"[It] Slice node affinity Tests SliceNodeAffinity tests - includes creation and deletion slice node affinity creation success test","historyId":"Controller Suite:Controller Suite#[It] Slice node affinity Tests SliceNodeAffinity tests - includes creation and deletion slice node affinity creation success test","time":{"start":1693899867000,"stop":1693899885859,"duration":18859},"status":"failed","statusMessage":"Unexpected error:\n    <*shell.ErrWithCmdOutput | 0xc000017aa0>: {\n        Underlying: <*exec.ExitError | 0xc00072f240>{\n            ProcessState: {\n                pid: 6878,\n                status: 256,\n                rusage: {\n                    Utime: {Sec: 0, Usec: 353409},\n                    Stime: {Sec: 0, Usec: 263177},\n                    Maxrss: 102340,\n                    Ixrss: 0,\n                    Idrss: 0,\n                    Isrss: 0,\n                    Minflt: 13236,\n                    Majflt: 16,\n                    Nswap: 0,\n                    Inblock: 13016,\n                    Oublock: 20624,\n                    Msgsnd: 0,\n                    Msgrcv: 0,\n                    Nsignals: 0,\n                    Nvcsw: 2442,\n                    Nivcsw: 3064,\n                },\n            },\n            Stderr: nil,\n        },\n        Output: {\n            stdout: {\n                Lines: nil,\n                merged: {\n                    Mutex: {state: 0, sema: 0},\n                    Lines: [\n                        \"Error from server (InternalError): error when creating \\\"/tmp/Slice%20node%20affinity%20Tests%20SliceNodeAffinity%20tests%20-%20includes%20creation%20and%20deletion%20slice%20node%20affinity%20creation%20success%20test962941551\\\": Internal error occurred: failed calling webhook \\\"msliceconfig.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-sliceconfig?timeout=10s\\\": dial tcp 10.96.230.189:443: connect: connection refused\",\n                    ],\n                },\n            },\n            stderr: {\n                Lines: [\n                    \"Error from server (InternalError): error when creating \\\"/tmp/Slice%20node%20affinity%20Tests%20SliceNodeAffinity%20tests%20-%20includes%20creation%20and%20deletion%20slice%20node%20affinity%20creation%20success%20test962941551\\\": Internal error occurred: failed calling webhook \\\"msliceconfig.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-sliceconfig?timeout=10s\\\": dial tcp 10.96.230.189:443: connect: connection refused\",\n                ],\n                merged: {\n                    Mutex: {state: 0, sema: 0},\n                    Lines: [\n                        \"Error from server (InternalError): error when creating \\\"/tmp/Slice%20node%20affinity%20Tests%20SliceNodeAffinity%20tests%20-%20includes%20creation%20and%20deletion%20slice%20node%20affinity%20creation%20success%20test962941551\\\": Internal error occurred: failed calling webhook \\\"msliceconfig.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-sliceconfig?timeout=10s\\\": dial tcp 10.96.230.189:443: connect: connection refused\",\n                    ],\n                },\n            },\n            merged: {\n                Mutex: {state: 0, sema: 0},\n                Lines: [\n                    \"Error from server (InternalError): error when creating \\\"/tmp/Slice%20node%20affinity%20Tests%20SliceNodeAffinity%20tests%20-%20includes%20creation%20and%20deletion%20slice%20node%20affinity%20creation%20success%20test962941551\\\": Internal error occurred: failed calling webhook \\\"msliceconfig.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-sliceconfig?timeout=10s\\\": dial tcp 10.96.230.189:443: connect: connection refused\",\n                ],\n            },\n        },\n    }\n    error while running command: exit status 1; Error from server (InternalError): error when creating \"/tmp/Slice%20node%20affinity%20Tests%20SliceNodeAffinity%20tests%20-%20includes%20creation%20and%20deletion%20slice%20node%20affinity%20creation%20success%20test962941551\": Internal error occurred: failed calling webhook \"msliceconfig.kb.io\": failed to call webhook: Post \"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-sliceconfig?timeout=10s\": dial tcp 10.96.230.189:443: connect: connection refused\noccurred","statusTrace":"/e2e/tests/controller/t_slicenodeaffinity_test.go:36\ngithub.com/kubeslice/kubeslice-e2e-automation-ent/tests/controller_test.glob..func5.1.1()\n\t/e2e/tests/controller/t_slicenodeaffinity_test.go:36 +0x1c3","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"steps":[{"name":"","time":{},"steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Report Entries:","time":{},"steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"By Step","time":{},"steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"/e2e/tests/controller/t_slicenodeaffinity_test.go:34","time":{},"steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"2023-09-05T08:15:10.941287581Z","time":{},"steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"&{Text:Deploying slicered Duration:0s}","time":{},"steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":6,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[],"labels":[{"name":"resultFormat","value":"junit"},{"name":"suite","value":"Controller Suite"},{"name":"testClass","value":"Controller Suite"},{"name":"package","value":"Controller Suite"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":2},"items":[{"uid":"198ca866a5f2163a","status":"passed","time":{"start":1693828311000,"stop":1693828333629,"duration":22629}}]},"tags":[]},"source":"72050e38879c10a3.json","parameterValues":[]}